account_invoice_change_currency/tests/test_account_invoice_change_currency.py

```diff
@@ -0,0 +1,310 @@
+# Copyright 2018 Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+import odoo.tests.common as common
+from odoo import fields
+from odoo.tools import float_compare
+
+
+class TestAccountInvoiceChangeCurrency(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+
+        self.precision = self.env["decimal.precision"].precision_get("Payment Terms")
+        res_users_account_manager = self.env.ref("account.group_account_manager")
+        self.manager = (
+            self.env["res.users"]
+            .with_context(no_reset_password=True)
+            .create(
+                dict(
```

Comment:
why not use the {} definition?

Reply:
@hugho-ad be aware if original is as it so any change related could be out of only migration

Reply:
I think this out of migration process before v14.0